### DISCUSSION TITLE
Что-то с диалогами?

### DISCUSSION THREAD

**Discussion by omqmo** [taiga-family/taiga-ui#5740] [Q&A] [answered]
Версия 3.50.0, над проектом работаю уже давно и сегодня потребовалось добавить новый диалог, который контент берет из компонента что-то типа 

```
this.dialogs.open(
    new PolymorpheusComponent( ConditionDeliveryDialogComponent, this.injector ));
```

но он не захотел работать. Оверлэй появляется, но в дом дереве нет указанного компонента и ошибок тоже нет. Но потупив немного я вдруг решил проверить старые диалоги и оказалось, что с ними точно такая же беда. Я вообще ничего связанного с ними не делал очень давно. Но совсем недавно просто открывал их и все работало. И мне даже сложно представить, что я мог бы сделать чтобы получить подобное повежденье. Вот скрин диалога кнтента просто вообще нет и нет ошибок. 

![Screenshot_2023-10-25_13-35-01](https://github.com/taiga-family/taiga-ui/assets/39467725/59c3aa08-96d7-48cf-8207-8aadf404ee0c)

Вопрос - вы что-то делали с диалогами? И чтобы мог сделать такого я, что и не работает и ошибки нет?

**Comment by waterplea** [comment, marked as answer]
Подозреваю, что затесалось 2 версии пакета `@tinkoff/ng-polymorpheus`. Попробуй следующее:

1. Удали `@tinkoff/ng-polymorpheus` из `package.json`, если он там есть
2. Удали `package-lock.json` и `node_modules`
3. Сделай чистый `npm i`

**Reply by omqmo** [reply]
Ооо, да. Заработало! Спасибище! 
node_modules удалить я и сам додумался, но вот про package-lock.json что-то вообще не подумал.
